Sass less stylus docs (#760)
parcel-bundler/website: src/languages/less.md

  order: 8
---

> Parcel supports [less](https://lesscss.org/) files out of the box by utilizing the `@parcel/transformer-less` plugin, under the hood this plugin uses the `less` npm package.

Reviewer: ```suggestion
Parcel supports [Less](https://lesscss.org/) files out of the box by utilizing the `@parcel/transformer-less` plugin, under the hood this plugin uses the `less` npm package.
```

## Example Usage

> Importing less in JavaScript/TypeScript

Reviewer: ```suggestion
Importing Less in JavaScript/TypeScript
```

```JS
import './custom.less'
```

> You can also directly include the less file in a HTML file.

Reviewer: ```suggestion
You can also directly include the Less file in a HTML file.
```

```HTML
<link rel="stylesheet" href="./style.less">
```

Directly compile less using the Parcel CLI

```

> parcel ./style.less

Reviewer: ```suggestion
parcel build ./style.less
```

would be more practical

  order: 8
---

Parcel supports [Less](https://lesscss.org/) files out of the box by utilizing the `@parcel/transformer-less` plugin, under the hood this plugin uses the `less` npm package.

> In the default Parcel config the [PostCSS](/languages/postcss) transformation pipeline also gets run over the compiled Less file to ensure minification and css modules just work.

Reviewer: ```suggestion
In the default Parcel config, the compiled Less files are also processed by [PostCSS](/languages/postcss) (meaning PostCSS plugins are executed and CSS modules just work: `import * as styles from "./style.module.less";`).
```
